src/typed-method-types/workflows/triggers/shortcut.ts

```diff
@@ -1,8 +1,26 @@
-import { BaseTrigger, TriggerTypes } from "./mod.ts";
+import { BaseResponse } from "../../../types.ts";
+import { BaseTriggerResponse } from "./base_response.ts";
+import { BaseTrigger, FailedTriggerResponse, TriggerTypes } from "./mod.ts";
 
-export type ShortcutTrigger = BaseTrigger & {
-  type: typeof TriggerTypes.Shortcut;
-  shortcut?: {
-    button_text: string;
+export type ShortcutTrigger =
+  & BaseTrigger
+  & {
+    type: typeof TriggerTypes.Shortcut;
+    shortcut?: {
+      button_text: string;
+    };
   };
-};
+
+export type ShortcutTriggerResponse<
+  TriggerDefinition extends ShortcutTrigger,
+> = Promise<
+  ShortcutResponse<TriggerDefinition> | FailedTriggerResponse
+>;
+export type ShortcutResponse<TriggerDefinition extends ShortcutTrigger> =
+  & BaseResponse
+  & {
+    trigger: ShortcutTriggerObject<TriggerDefinition>;
+  };
+export type ShortcutTriggerObject<TriggerDefinition extends ShortcutTrigger> =
```

Comment:
Shouldn't this also return the `ShortcutTrigger` itself so that it includes the `shortcut` object and `button_text` if provided?

Reply:
the response to a shortcut doesn't contain a shortcut object, even if one was provided. Checked against both the console in the hermes app and against the api tester.